.ci/travis.sh

```diff
@@ -139,7 +142,13 @@ jacoco)
   ;;
 
 quarterly-cache-cleanup)
-  find ~/.m2 -maxdepth 4 -type d -mtime +90 -exec rm -rf {} \;
+  MVN_REPO=$(mvn -e --no-transfer-progress help:evaluate -Dexpression=settings.localRepository \
+    -q -DforceStdout);
+  if [[ -d ${MVN_REPO} ]]; then
+    find "$MVN_REPO" -maxdepth 4 -type d -mtime +90 -exec rm -rf {} \;
```

Comment:
I would prefer a simply appending `|| true`, if cache is where we expect, OK. If not, download dependencies and run maven. Adding another maven command just adds fragility and makes for a new point of failure.

Reply:
if `find` errors out, will it display that with `|| true`? Also do you want me to remove the display from the init too?

Reply:
>if find errors out, will it display that with || true?

yes:
```
➜  checkstyle git:(fix-link-check) find notpresent -maxdepth 4 -type d -mtime +90 -exec rm -rf {} \;        

find: ‘notpresent’: No such file or directory
➜  checkstyle git:(fix-link-check) echo $?
1
➜  checkstyle git:(fix-link-check) find notpresent -maxdepth 4 -type d -mtime +90 -exec rm -rf {} \; || true

find: ‘notpresent’: No such file or directory
➜  checkstyle git:(fix-link-check) echo $?                                                                  
0

```

If build is failing, what value will location of maven repo provide?

Reply:
It will tell us if maven repo changes location, which would be one cause for the failure.

Reply:
I would prefer to keep scripts as simple as possible, and add debugging commands as needed when a problem arises. But let's get a second opinion. @romani ?

Reply:
I always prefer simplicity of bash scripts. I agree with Nick.
But extra logging is also ok.